crates/autopilot/src/infra/solvers/dto/settle.rs

```diff
@@ -11,6 +11,9 @@ pub struct Request {
     /// Unique ID of the solution (per driver competition), to settle.
     #[serde_as(as = "serde_with::DisplayFromStr")]
     pub solution_id: u64,
+    /// The last block number in which the solution TX can be included
+    #[serde_as(as = "serde_with::DisplayFromStr")]
+    pub block_deadline: u64,
```

Comment:
Seems more descriptive.
```suggestion
    pub submission_deadline: u64,
```